src/lightning_lite/accelerators/cpu.py

```diff
@@ -0,0 +1,91 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Dict, List, Union
+
+import torch
+
+from lightning_lite.accelerators.accelerator import Accelerator
+from lightning_lite.utilities import device_parser
+from lightning_lite.utilities.imports import _PSUTIL_AVAILABLE
+
+
+class CPUAccelerator(Accelerator):
+    """Accelerator for CPU devices."""
+
+    def init_device(self, device: torch.device) -> None:
+        """
+        Raises:
+            ValueError:
+                If the selected device is not CPU.
+        """
+        if device.type != "cpu":
+            raise ValueError(f"Device should be CPU, got {device} instead.")
+
+    def get_device_stats(self, device: torch.device) -> Dict[str, Any]:
+        """Get CPU stats from ``psutil`` package."""
+        return get_cpu_stats()
+
+    def teardown(self) -> None:
+        pass
+
+    @staticmethod
+    def parse_devices(devices: Union[int, str, List[int]]) -> int:
+        """Accelerator device parsing logic."""
+        devices = device_parser.parse_cpu_cores(devices)
```

Comment:
(maybe not in this PR): I would move these utility functions out of `device_parser` into the accelerator that uses them.

I understand we keep them because the PL accelerator will want to use them too, but PL can import them from this Lite module. Code locality is good! Say no to random utility files!

Reply:
Yes